events/forms.py

```diff
@@ -679,16 +681,22 @@ def __init__(self):
             exclude=('send_survey',)
         )
 
+        cancelled_reason_edit = FieldAccessLevel(
+            lambda user, instance: user.has_perm('events.event_view_sensitive') and \
```

Comment:
Is there a reason why this requires sensitive events?

Reply:
Moved to `events.cancel_event`